tsconfig.json

```diff
@@ -33,7 +33,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "strict": true,
-    "typeRoots": ["./src/types/*.d.ts"]
+    "typeRoots": ["./src/types/*.d.ts", "./node_modules/@types"],
```

Comment:
I found it quite interesting that we did not already get the types from the modules. 

Weird that we haven't gotten issues before, but maybe this is the first time we explicitly need the type from a module.